### PR TITLE
Implement bogo in ruby

### PR DESCRIPTION
**Pull request by jasmaa**

### CODE REVIEW

File: contents/bogo_sort/code/ruby/bogo.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+def is_sorted(a)
+  for i in 0...a.length-1
```
Comment (jiegillet) [inline contents/bogo_sort/code/ruby/bogo.rb]: It should be `a.length-2` right?

File: contents/bogo_sort/code/ruby/bogo.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+def bogo_sort(a)
+  while !is_sorted(a)
+    a.shuffle!
+  end
+end
+
+def main()
+	a = [1, 1, 0, 3, 7]
+	bogo_sort(a)
+	print a
```
Comment (jiegillet) [inline contents/bogo_sort/code/ruby/bogo.rb]: Print something like "Unsorted: " then the array, then "Sorted:"

File: contents/bogo_sort/code/ruby/bogo.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+end
+
+main()
```
Comment (jiegillet) [inline contents/bogo_sort/code/ruby/bogo.rb]: I think the file should end with a newline

File: contents/bogo_sort/code/ruby/bogo.rb (hunk 1 above)
Comment (jasmaa) [reply contents/bogo_sort/code/ruby/bogo.rb]: The range is exclusive and it runs up to the penultimate element so it should be fine I think?...

Comment (jiegillet) [reply contents/bogo_sort/code/ruby/bogo.rb]: Ah, sorry `...` is exclusive, you're right then!

**Pull request merged**